README.md

```diff
@@ -65,15 +65,14 @@ place after an MSC has been accepted, not as part of a proposal itself.
    Alternatively, use the Docker image at
    https://hub.docker.com/r/klakegg/hugo/. (The "extended edition" is required
    to process the SCSS.)
-2. Run `git submodule update --init --recursive` for good measure.
```

Comment:
do we not still need a `git submodule update --init` here, even if `--recursive` is no longer necessary? docsy is still a submodule which needs cloning/